### ISSUE TITLE
Update CoP: Ops: Add and Remove leaders

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#7497]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_data/internal/communities/ops.yml` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x] Find the `leadership` variable and replace. 
```
leadership:
  - name: Chelsey Beck
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/U01J93AQKSS
      github: https://github.com/chelseybeck
    picture: https://avatars.githubusercontent.com/chelseybeck
  - name: Ernie Pelayo
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/U046MDSM3PA
      github: https://github.com/erniep278
    picture: https://avatars.githubusercontent.com/erniep278
  - name: Dean Church
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/UJR0UKBN1
      github: https://github.com/Rankazze
    picture: https://avatars.githubusercontent.com/Rankazze
  - name: Nayan Bhatt
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/U05TZLLJAUV
      github: https://github.com/freaky4wrld
    picture: https://avatars.githubusercontent.com/freaky4wrld
  - name: Zoey Zhang
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/U06DT2CA92T
      github: https://github.com/abbyz123
    picture: https://avatars.githubusercontent.com/abbyz123
```
with this
```
leadership:
  - name: Chelsey Beck
    github-handle: chelseybeck
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/U01J93AQKSS
      github: https://github.com/chelseybeck
    picture: https://avatars.githubusercontent.com/chelseybeck
  - name: Alex English
    github-handle: ale210
    role: Co-lead
    links:
      slack: U078B1S0XDM
      github: https://github.com/ale210
    picture: https://avatars.githubusercontent.com/ale210
  - name: Ryan Sakuma
    github-handle: RSkuma
    role: Co-lead
    links:
      slack: U078U294XB6
      github: https://github.com/RSkuma
    picture: https://avatars.githubusercontent.com/RSkuma
  - name: Sudha Raamakrishnan
    github-handle: sudhara
    role: Ops PM
    links:
      slack: https://hackforla.slack.com/team/U07FJRNN9MK
      github: https://github.com/sudhara
    picture: https://avatars.githubusercontent.com/sudhara
  - name: Dean Church
    github-handle: Rankazze
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/UJR0UKBN1
      github: https://github.com/Rankazze
    picture: https://avatars.githubusercontent.com/Rankazze
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] [Insert name of project] page [^2]
- [x] Once your pull request is merged, go to the initiating ER [^3]
  - [x] Check off this issue under the _Dependency_ section
  - [x] move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label** and add the `ready for stakeholder` label
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/communities-of-practice
[^3]: initiating ER:  #7294

**Comment by HackforLABot** [comment]
Hi @NolaDodd, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by NolaDodd** [comment]
i. Availability: Weekdays 10am to 5pm
ii. ETA: Today or Tomorrow (Thursday 9/19; Friday 9/20).
